Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/LocalStorageContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace MvvmSeed.Domain.Model
+{
+    public class LocalStorageContext : DbContext
+    {
+        public DbSet<RandomizedString> RandomizedStrings { get; set; }
+
+        private string DatabasePath { get; set; }
```

Comment:
So I guess you are going to assume local storage is always a database instead of a file, memory or whatever... right? Asking because the names you are using. I remember when you proposed me to have something agnostic instead.